### ISSUE TITLE
The linked blog post has moved

### ISSUE THREAD

**Issue by billgraziano** [MicrosoftDocs/sql-docs#1881]
it should probably be removed.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 40178a54-5c50-6667-f605-f6c0d75f8741
* Version Independent ID: 9a0b755c-40a0-563a-b945-c0216ae71f7b
* Content: [Columnstore indexes - Data loading guidance - SQL Server](https://docs.microsoft.com/en-us/sql/relational-databases/indexes/columnstore-indexes-data-loading-guidance?view=sql-server-2017#feedback)
* Content Source: [docs/relational-databases/indexes/columnstore-indexes-data-loading-guidance.md](https://github.com/MicrosoftDocs/sql-docs/blob/live/docs/relational-databases/indexes/columnstore-indexes-data-loading-guidance.md)
* Product: **sql**
* Technology: **table-view-index**
* GitHub Login: @MikeRayMSFT
* Microsoft Alias: **mikeray**

**Comment by julieMSFT** [comment]
@billgraziano 
thanks for bringing this to our attention! Let me look into this a bit further. Thanks!

**Comment by MightyPen** [comment]
I am fixing this for Mike, with my PR 9812.
https://github.com/MicrosoftDocs/sql-docs-pr/pull/9812

I was able to find the old blog post as it was migrated to techcommunity.microsoft.com .
I did a Bing search on the title-ish portion of the old MSDN https URL, to find the new address.
The link was and still is at the very bottom of the article.

Closing now this Issue 1881, as Fixed.

CC: @MikeRayMSFT
